scala-package/core/src/main/scala/org/apache/mxnet/NDArray.scala

```diff
@@ -407,11 +407,30 @@ object NDArray extends NDArrayBase {
    * @param dType The data type of the `NDArray`. The default datatype is `DType.Float32`.
    * @return NDArray of evenly spaced values in the specified range.
    */
-  def arange(start: Float, stop: Option[Float] = None, step: Float = 1.0f,
+  def arange(start: Float, stop: Option[Float] = None, step: Float = 1.0f, repeat: Int = 1, 
```

Comment:
@taliesinb Thanks for making this change. There is a compile error and CI is breaking. 
I also want to slightly change this, I fixed the compile error and modified NDArray/Symbol and pushed a commit to your branch

Reply:
@lanking520 FYI

Reply:
@nswamy oh thanks! my bad.

Reply:
@nswamy Looks good!